config/default.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
default.json should have 'default' values (that can be hardcoded strings)

Reply:
Done